prijateli_tree/app/routers/games.py

```diff
@@ -484,6 +484,67 @@ def real_game_transition(
     return templates.TemplateResponse("real_game_transition.html", result)
 
 
+@router.get(
+    "/{game_id}/player/{player_id}/end_of_session",
+    response_class=HTMLResponse,
+)
+def end_of_session(
+    request: Request,
+    game_id: int,
+    player_id: int,
+    db: Session = Depends(get_db),
+) -> Response:
+    """
+    Function that returns the end of session page and
+    template.
+    """
+    game, player = get_game_and_player(game_id, player_id, db)
+    session_player = get_session_player_from_player(player, db)
+
+    # Get points and won games from session player
+    total_points = session_player.points
+    won_games = session_player.correct_answers
+    denars = int(total_points * DENAR_FACTOR)
+
+    template_text = languages[get_lang_from_player_id(player_id, db)]
+
+    result = {
+        "request": request,
+        "game_id": game_id,
+        "total_points": total_points,
+        "won_games": won_games,
+        "text": template_text,
+        "denars": denars,
+        "player_id": player_id,
+        "game_id": game_id,
+    }
+
+    return templates.TemplateResponse("end_of_session.html", result)
+
+
+@router.get(
+    "/{game_id}/player/{player_id}/thank_you",
+    response_class=HTMLResponse,
+)
+def thank_you(
+    request: Request,
+    game_id: int,
+    player_id: int,
+    db: Session = Depends(get_db),
+) -> Response:
+    """
+    Sends player to thank you page
+    """
+    template_text = languages[get_lang_from_player_id(player_id, db)]
+    result = {
+        "request": request,
+        "player_id": player_id,
+        "game_id": game_id,
+        "text": template_text,
+    }
```

Comment:
Do we need the player_id and game_id back to the template? I don't think so, so let's not.